homeassistant/components/openweathermap/weather.py

```diff
@@ -100,6 +115,17 @@ def __init__(
         elif mode == OWM_MODE_FREE_FORECAST:
             self._attr_supported_features = WeatherEntityFeature.FORECAST_HOURLY
 
+    async def async_get_minute_forecast(self) -> None:
```

Comment:
return typing is incorrect, we do return something

Reply:
Return typing corrected